src/analytics/addPageAction.ts

```diff
@@ -0,0 +1,47 @@
+// @ts-expect-error
+import type { NewRelicAPI } from "new-relic-browser";
+import { parseQueryString } from "utils/query-string";
+
+declare global {
+  interface Window {
+    newrelic: typeof NewRelicAPI; // eslint-disable-line no-undef
+  }
+}
+
```

Comment:
I was reading the [forums](https://discuss.newrelic.com/t/typescript-for-browser-agent/35942/16) and it seems like this stuff isn't needed after the updates to `@types/new-relic-browser` (check the PR in the link for more details). 

So it seems like you don't even need to import anything from `new-relic-browser` anymore, but `newrelic` is just globally available now? It's working on my end but give it a try 😅
```suggestion
import { parseQueryString } from "utils/query-string";
```